spec/spec_helper.rb

```diff
@@ -22,4 +25,11 @@
   config.expect_with(:rspec) do |c|
     c.syntax = %i[should expect]
   end
+
+  config.around do |example|
```

Comment:
All good to have this in `spec_helper.rb` and apply to all specs. Even 10 seconds is extremely generous.